types/rdfjs__data-model/rdfjs__data-model-tests.ts

```diff
@@ -0,0 +1,48 @@
+import factory from "@rdfjs/data-model";
+import * as RDF from "@rdfjs/types";
+
+const myQuad = factory.quad(
+  factory.namedNode('http://example.org/subject'),
+  factory.namedNode('http://example.org/predicate'),
+  factory.namedNode('http://example.org/object'),
+);
+
+const myQuadWithLiteral = factory.quad(
+  factory.namedNode('http://example.org/subject'),
+  factory.namedNode('http://example.org/predicate'),
+  factory.literal('34'),
+);
+
+const myQuadWithVariable = factory.quad(
+  factory.namedNode('http://example.org/subject'),
+  factory.namedNode('http://example.org/predicate'),
+  factory.variable('?o'),
+);
+
+const myQuadWithGraph = factory.quad(
+  factory.namedNode('http://example.org/subject'),
+  factory.namedNode('http://example.org/predicate'),
+  factory.namedNode('http://example.org/object'),
+  factory.defaultGraph()
+);
+
+const myBaseQuad = factory.quad<RDF.BaseQuad>(
+  factory.namedNode('http://example.org/subject'),
+  factory.blankNode('34'),
+  factory.namedNode('http://example.org/object'),
+);
+
+const myBaseQuadBad = factory.quad(
+  factory.namedNode('http://example.org/subject'),
+  // @ts-expect-error
```

Comment:
```suggestion
  // $ExpectError
```
why not this? (in DT tests context)

Reply:
Didn't know DT had this, you learn something every day, thanks!